frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -24,6 +24,13 @@ const useStyles = makeStyles({
     width: '100%',
     padding: '17px',
   },
+  imgMobile: {
+    borderRadius: '8%',
+    height: '480px',
```

Comment:
It's ok to set the height for the non-mobile image because that stops the image from getting smaller vertically, but let's not set the height on the mobile image. If you display it on a phone right now, the cards are too long: 
<img width="276" alt="Screen Shot 2022-05-02 at 1 12 37 AM" src="https://user-images.githubusercontent.com/57203639/166187359-ef790aef-3fa3-494c-819b-85e4ba3cc7e9.png">

If you just get rid of `height: '480px'`, it looks like this: 
<img width="257" alt="Screen Shot 2022-05-02 at 1 14 30 AM" src="https://user-images.githubusercontent.com/57203639/166187428-c842ad4e-53ce-459c-8cef-c3305a6e093d.png">

Reply:
Thanks! I just made the adjustment!